platform/macos/os_macos.mm

```diff
@@ -341,11 +341,11 @@
 
 	String font_name = p_font_name;
 	if (font_name.to_lower() == "sans-serif") {
-		font_name = "Helvetica";
+		font_name = "SF Pro";
 	} else if (font_name.to_lower() == "serif") {
-		font_name = "Times";
+		font_name = "New York";
 	} else if (font_name.to_lower() == "monospace") {
-		font_name = "Courier";
+		font_name = "SF Mono";
```

Comment:
None of these fonts exist in macOS 13.0.1.

List of preinstalled fonts https://developer.apple.com/fonts/system-fonts/#preinstalled

Reply:
There are similar, hidden (with a dot in the beginning of the name) system fonts that can be selected `.SF NS`, `.SF NS Mono`, `.New York` that are preinstalled. But names are not the same in the different versions of macOS (10.14 and older have `.SF NS Display` and no New York or Mono).